samples/net/zperf/sample.yaml

```diff
@@ -38,3 +38,8 @@ tests:
       - CONFIG_SHELL_HISTORY=n
     tags: usb net zperf
     depends_on: usb_device
+  sample.net.zperf.shield:
+    platform_whitelist: reel_board
```

Comment:
I think we shouldn't need  `platform_whitelist`. Filtering on `arduino_spi` shouldbe sufficient and I think that @nashif wanted to clean those.

Reply:
Sanitycheck does not pick reel board without `platform_whitelist` because tests are not enabled for this board per default.